### PR TITLE
Connect Local storage

### PR DESCRIPTION
**Pull request by J-Bannach**
- Write hook to store data in local storage
- Add write function for local storage: Entries from local storage are now saved on home page

### CODE REVIEW

File: src/app/components/MealSelection/MealSelection.module.css [hunk 1]
```diff
@@ -6,4 +6,8 @@
   color: var(--text-color);
   border-bottom: var(--text-color) 0.01em solid;
   text-decoration: none;
+  width: 20em;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (MaxMuecket) [inline src/app/components/MealSelection/MealSelection.module.css:12]: I learned new CSS variables. Thanks! ✌️

File: src/utils/parseFromLocalStorage.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
+type DayTimeDishItem = {
```
Comment (MaxMuecket) [inline src/utils/parseFromLocalStorage.ts:1]: Maybe you can add these types to your external types.ts, but I'm not sure 😬

**Review by MaxMuecket — APPROVED** [review]
I like your coding style! ✌️

File: src/utils/parseFromLocalStorage.ts (hunk 2 above)
Comment (J-Bannach) [reply src/utils/parseFromLocalStorage.ts:1]: Good point! Will do some cleaning up after the basic functions are working 😃

**Pull request merged**